llama_stack/distribution/datatypes.py

```diff
@@ -117,6 +117,21 @@ class Provider(BaseModel):
     config: Dict[str, Any]
 
 
+class ServerConfig(BaseModel):
+    port: int = Field(
+        default=8321,
```

Comment:
Does this require a doc update? I believe we’re currently serving it on port 8000. Let me know if that’s still accurate!

Reply:
I believe we updated things to 8321 in our 0.1.0 update. It was 5000 before.